### PR TITLE
Milestone 4 & 5 - Styling and captions

### PR DESCRIPTION
**Pull request by abdulrauf11**
The PR provides a build with the first pass on the styling of the application and demonstrates a fully functional Quiz Screen.

## Tasks
- All the screens have been styled according to the Figma link.
- Captions are also being used on all the screens (where they are required)

## To Do
- Sound effects used throughout the application
- Update with final assets
- Code cleanup and optimization
- Some edge cases for the quiz

### Edge cases
1. If all players enter answers before audio narration completes, the results are shown as soon as the audio narration completes.
2. When appropriate, the media asset updates to display the "reveal" answer version.
3. Inactivity timeout - if no answers are entered for two quiz questions in a row, the application reloads/resets to the Attract screen.
4. Software must be able to run in an exhibit for ~12 hours without any performance issues. You may reload the page programmatically after a period of inactivity to help avoid memory issues.

Let me know if we have missed an edge case.

## Test Link
https://tsck-bionic-eyes-quiz-show.netlify.app/

## Preview
Screenshot 2021-09-22 at 3 31 26 PM Screenshot 2021-09-22 at 3 31 42 PM Screenshot 2021-09-22 at 3 32 17 PM Screenshot 2021-09-22 at 3 32 47 PM

### CODE REVIEW

File: src/styles/reset.scss [hunk 1]
```diff
@@ -0,0 +1,133 @@
+html,
```
Comment (tnordberg) [inline src/styles/reset.scss:1]: Interesting! I haven't seen this before but I see the utility after Googling:
https://meyerweb.com/eric/tools/css/reset/

**Review by tnordberg — APPROVED** [review]
This milestone required `Fully functional app with minimal styling` which is easily represented in this P.R. All the styling work so far looks accurate and crisp. Excellent work.

A few requests to include in the next milestone:

1. Hide/disable all UI controls from all video players: https://d.pr/i/K8I919
2. Players should receive 1 point per correct answer, not 5. 
3. When timer reaches zero, please dim by reducing opacity to 30%. Like this: https://d.pr/i/xWc8zm

I will follow up in chat about where we are at with the caption files, but consider this milestone approved.
